src/agent/agent.c

```diff
@@ -1334,6 +1334,27 @@ static int agent_method_disable_metrics(sd_bus_message *m, void *userdata, UNUSE
         return sd_bus_reply_method_return(m, "");
 }
 
+/*************************************************************************
+ ************** org.containers.hirte.Agent.SetNodeLogLevel ***************
+ ************************************************************************/
+
+static int agent_method_set_log_level(
+                UNUSED sd_bus_message *m, UNUSED void *userdata, UNUSED sd_bus_error *ret_error) {
+        const char *level = NULL;
+        int r = sd_bus_message_read(m, "s", &level);
+        if (r < 0) {
+                return sd_bus_reply_method_errorf(
+                                m, SD_BUS_ERROR_FAILED, "Failed to read the parameter: %s", strerror(-r));
```

Comment:
Same as in manager, wouldn't it be good to mention which parameter failed to be read?